docs/HowTo/Configure/Configure-HA/Sample-Configuration.md

```diff
@@ -0,0 +1,94 @@
+description: Sample Load Balancers 
+<!--- END of page meta data -->
+
+# Sample Load Balancer Configurations 
+
+## AWS 
+
+For AWS, we recommend the Classic Load Balancer. The Classic Load Balancer routes layer 4 and 7. Register
+the Hyperledger Besu instances to the load balancer and use the [liveness endpoint](../../Interact/APIs/Using-JSON-RPC-API.md#readiness-and-liveness-endpoints)
+for health checks. 
+
+Another alterntive is the Application Load Balancer:
+ 
+* Configure 1 target group with n nodes 
+* Use the [liveness endpoint](../../Interact/APIs/Using-JSON-RPC-API.md#readiness-and-liveness-endpoints) for health checks
+* Register the Besu instances multiple times with different ports. Similar to configuring microservices 
+on Elastic Container Service (ECS) or Elastic Kubernetes Service (EKS) 
+* Configure multiple listeners with one per port you're using and route to the target group. 
```

Comment:
Move this bullet point to be the second one. Rename `one per port` to `one per port (eg: 30303, 8545 etc)`

Reply:
Done